### ISSUE TITLE
Compass Manager - Cutover plan for STAGE and PROD rollout

### ISSUE THREAD

**Issue by tobiscr** [kyma-project/compass-manager#80]
**Description**

To ensure a controlled rollout of the Compass Manager, this cut-over plan is used to define the required steps, their order and owner:

- [x] #28 
- [x] Dry-Run on STAGE completed and results reviewed, Owner: SRE / @kyma-project/framefrog 
- [x] Enabling Compass Manager on STAGE,Owner: SRE
- [x] Review with SRE required steps for PROD rollout,  Owner: SRE / @kyma-project/framefrog 

**Requires**

#87 
https://github.com/kyma-project/control-plane/issues/3088

**More information**
Please follow the migration plan prepared for the process. 

**Rollout workplan:** 

DEV environemnt
- [x] Compass-manager is deployed and  registers SKRs in Compass when application-connector module is enabled
- [x] Modified Provisioner is deployed
- [x] Run migration application to Generate CompassMappings for existing provisioned and connected runtimes
- [x] Modify Provisioner overrides to disable integration with Compass

STAGE environemnt
- [x] Compass-manager is deployed and  registers SKRs in Compass when application-connector module is enabled
- [x] Modified Provisioner is deployed
- [x] SRE deploys compass-manager on stage
- [x] SRE runs migration application to Generate CompassMappings for existing provisioned and connected runtimes
- [x] Modify Provisioner overrides to disable integration with Compass

PROD environemnt 
- [x] Compass-manager is deployed and  registers SKRs in Compass when application-connector module is enabled
- [x] Microdelivery Issue is accepted 
- [x] Missing Component Requirements are delivered 
    - [x] Backup Requirements (#backlog/5354) 
    - [x] RBAC report (#backlog/5374)
- [x] Modified Provisioner is deployed
- [x] Run migration application to Generate CompassMappings for existing provisioned and connected runtimes (#backlog/5345)
- [x] Modify Provisioner overrides to disable integration with Compass
- [x] Manual test of Application Integration with Kyma 

Stretch: 

- [ ] - Fully remove Compass Integration code from Provisioner https://github.com/kyma-project/control-plane/pull/3411

**Comment by mvshao** [comment]
Before STAGE and PROD rollout we need to migrate old runtimes that are connected to the Compass and don't have the Application Connector module enabled (creation of the Compass Manager Mapping CRs for already registered runtimes)

**Comment by tobiscr** [comment]
depends on #87

**Comment by zhoujing2022** [comment]
@koala7659 as we discussed please let me know if the tasks are done before noon of 10th.Jan. thx.

**Comment by zhoujing2022** [comment]
Compass mgr. onto STAGE it’s postponed to 18th.Jan.2024
Compass mgr. onto  PRODUCTION it’s postponed to 25th.Jan.2024

**Comment by Sawthis** [comment]
Compass mgr. onto STAGE: it’s postponed to 15th.Mar.2024
Compass mgr. onto PRODUCTION: it’s postponed to 22nd.Mar.2024

**Comment by koala7659** [comment]
Unfortunately we must postpone STAGE delivery again. 
After we disabled Compass Registration for Provisioner on KCP DEV we discovered that installation of Kyma Runtimes started to fail. We decided to revert the flags, and we must fix Provisioner now. The fix is not a big but we will need at least two days to deliver.

**Comment by koala7659** [comment]
Fix for Provisioner not creating `istio-system` namespace when Compass Registration is disabled https://github.com/kyma-project/control-plane/pull/3415

**Comment by koala7659** [comment]
Change request issue for the process https://github.tools.sap/kyma/backlog/issues/5345
